src/components/image-control2/index.js

```diff
@@ -83,6 +83,7 @@ const ImageControl = memo( props => {
 	return (
 		<AdvancedControl
 			{ ...controlProps }
+			suffix="Url"
```

Comment:
I understand why you need this. However I don't think this is the ideal solution because if you don't know the inner workings of the `AdvancedControl` component, you will not know what the `suffix` prop is for (that it's solely used for checking whether there's a value). I recommend changing this.

Let's go back to why you need this new prop, you need it so that the control will know whether the attribute has a value or not. Usually, the `prop.attribute` that's passed here is enough, but you need need to check a different attribute for images. So why not instead of a `suffix` prop, add a new `valueCheckAttribute` prop that's a string. Then inside the `AdvancedControl` component, instead of appending a suffix to the attribute, just use `valueCheckAttribute` if it's given.

You can pass it like this:

```jsx
valueCheckAttribute={ props.attribute + 'Url' }
```

In the `defaultProps` for `AdvancedControl` make sure to add comments so we can document what this prop does.